libbeat/docs/dashboards.asciidoc

```diff
@@ -102,6 +102,10 @@ predefined +{beatname_lc}-*+ index pattern and set it as the default.
 
 image:./images/kibana-created-indexes.png[Kibana configured indexes]
 
+NOTE: After importing the index-pattern in Kibana, you need to refresh each index-pattern in Kibana in order to update
```

Comment:
@monicasarbu I think this note will be confusing to new users. It would be better, as @ruflin suggests, to describe how the problem affects the user (what they see when they try to visualize the data) and how to fix the problem. I'd suggest having a short note here and then pointing off to an FAQ topic for more detail (in the FAQ topic, you could have a more detailed example of the problem).

BTW, when you say that they need to refresh the index pattern, do you mean that they need to do this: https://www.elastic.co/guide/en/kibana/current/settings.html#reload-fields? If so, a pointer to the Kibana doc about this would be useful.